.gitignore

```diff
@@ -1,2 +1,2 @@
 /vendor/
-/azure-devops-exporter
```

Comment:
please add *.exe below the line

Reply:
There is no `/azure-devops-exporter` folder in the repo and it is not created during build. Why is that line needed?